endpoints/user/save_user.php

```diff
@@ -125,7 +125,7 @@ function update_exchange_rate($db) {
         if ($result) {
             $cookieExpire = time() + (30 * 24 * 60 * 60);
             $oldLanguage = isset($_COOKIE['language']) ? $_COOKIE['language'] : "en";
-            setcookie('language', $language, $cookieExpire, '/');
+            setcookie('language', $language, $cookieExpire);
```

Comment:
This will create the cookie with the path `/subdirectory/endpoints/user`

This would work:

```
$root = str_replace('/endpoints/user', '', dirname($_SERVER['PHP_SELF']));
$root = $root == '' ? '/' : $root;
setcookie('language', $language, $cookieExpire, $root);
```